### DISCUSSION TITLE
[react native] 안드로이드 빌드를 해보아요

### DISCUSSION THREAD

**Discussion by pipisebastian** [BDD-CLUB/challenge-every-week-posting#65] [1. 개발 관련 글 모음]
[블로그 ](https://velog.io/@pipi/react-native-%EC%95%88%EB%93%9C%EB%A1%9C%EC%9D%B4%EB%93%9C-apk-%EB%B9%8C%EB%93%9C%ED%95%98%EA%B8%B0vscode)링크입니다~

**Comment by lcqff** [comment]
헐~ 요새 리엑트 네이티브 공부하시네요!! 나중에 저도 맛보기 해볼때 참고하겠습니다~~!
